platform/spire/src/configuration.py

```diff
@@ -1,3 +1,4 @@
+from ipaddress import IPv4Address, IPv4Network
```

Comment:
w/r/t the commit message: I've been trying to avoid putting actual issue #'s in my commit messages, because it wreaks havoc on GitHub's mention tracking when you keep rebasing commits. (See #307's history for what I'm talking about.)

Reply:
oh no